pgo/cmd/create.go

```diff
@@ -368,6 +370,8 @@ func init() {
 	createClusterCmd.Flags().StringVarP(&BackrestS3Region, "pgbackrest-s3-region", "", "",
 		"The AWS S3 region that should be utilized for the cluster when the \"s3\" "+
 			"storage type is enabled for pgBackRest.")
+	createClusterCmd.Flags().StringVarP(&BackrestS3URIStyle, "pgbackrest-s3-uri-style", "", "", "Specifies whether \"host\" or \"path\" style URIs will be used when connecting to S3.")
+	createClusterCmd.Flags().BoolVarP(&BackrestS3VerifyTLS, "pgbackrest-s3-verify-tls", "", true, "Set this value to true to enable TLS verification when making a pgBackRest connection to S3. Defaults to false.")
```

Comment:
`This sets if pgBackRest should verify the TLS certificate when connecting to S3. Defaults to true. To disable, use --pgbackrest-s3-verify-tls=false`

Reply:
updated. I put the command in quotes and removed "Defaults to true." because the client output already includes that.
Output below:
```
--pgbackrest-s3-uri-style string        Specifies whether "host" or "path" style URIs will be used when connecting to S3.
--pgbackrest-s3-verify-tls              This sets if pgBackRest should verify the TLS certificate when connecting to S3. To disable, use "--pgbackrest-s3-verify-tls=false". (default true)
```